### ISSUE TITLE
Ability to create/update entries in Vault

### ISSUE THREAD

**Issue by matthewmrichter** [nyxcharon/vault-ui#32]
Maybe this is implied but I figured I would put in an issue so it can be tracked.. but this tool would be much more useful if we could create or update secrets once authenticated as well.

**Comment by nyxcharon** [comment]
Yeah, this is a planned feature for sure but i've been waiting to finish #11 first.

**Comment by amosshapira** [comment]
I'm interested in this UI to see if it can help us let our internal users self-manage their OpenVPN SSL certificates.
